cognite_toolkit/_cdf.py

```diff
@@ -91,7 +91,9 @@
     )
 
 default_typer_kws = dict(
-    pretty_exceptions_short=False, pretty_exceptions_show_locals=False, pretty_exceptions_enable=False
+    pretty_exceptions_short=False,
+    pretty_exceptions_show_locals=False,
+    pretty_exceptions_enable=False,
```

Comment:
It seems you have some extra automatic black linting on in your IDE that has a config we don't use in Toolkit. 

This clutters your PR and make it harder to read, so I would ideally like you to turn it off.

Reply:
understandable, but that's tricky for me atm :/
- using black-formatter on save, which I thought is best-practice 
  - but which tends to reformat oneliners as I can see
- applying `pre-commit run` does not fix the format as wanted by the repo

I know you use pycharm, but which local formatter do you use?

Reply:
deactivated my local `black` formatter in vscode until we find a solution

Reply:
I don't use autoformatter in my IDE. Instead, I just allow the local pre-commit linting in each project I work determine the formatting. 

I have earlier experimented with having an autoformatter on save, and stopped for this exact reason.